### PR TITLE
Action Creator 6: Allow user-defined options for select and radio inputs

### PR DESCRIPTION
**Pull request by iethree**
## Description

Allow users to define custom options for select and radio form inputs. It's not as pretty as the figma designs for now, but it keeps us moving forward.

## Screenshots

Screen Shot 2022-09-13 at 1 05 36 PM
Screen Shot 2022-09-13 at 1 04 59 PM

### CODE REVIEW

File: frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts [hunk 1]
```diff
@@ -40,7 +40,11 @@ const getSampleOptions = () => [
   { name: t`Option Three`, value: 3 },
 ];
 
+const getOptionsFromArray = (options: (number | string)[]) =>
+  options.map(o => ({ name: o, value: o }));
+
 const getParameterFieldProps = (fieldSettings: FieldSettings) => {
+  console.log(fieldSettings);
```
Comment (kulyk) [inline frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts:47]: We should remove it

**Review by kulyk — APPROVED** [review]
Nice!

File: frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts (hunk 1 above)
Comment (iethree) [reply frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts:47]: 👍  🤦

Comment (iethree) [inline frontend/src/metabase/writeback/components/ActionCreator/FormCreator/utils.ts:47]:
```suggestion
```

**Pull request merged**